packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -0,0 +1,73 @@
+# psammead-storybook-helpers - [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-storybook-helpers.svg)](https://www.npmjs.com/package/@bbc/psammead-storybook-helpers) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+This package provides a collection of common values that are used in storybook by the Psammead components.
+
+## Exports
+
+`LANGUAGE_VARIANTS` - A list of text samples in different languages.
+
+`inputProvider` - A function that provides support for selecting between the `text` and `select` storybook knobs. When using the `select` knob, users will be prompted to select a language, which will insert text from `LANGUAGE_VARIANTS` into the story.
+
+## Installation
+
+```sh
+npm install @bbc/psammead-storybook-helpers --save-dev
+```
+
+## Usage
+
+### LANGUAGE_VARIANTS
+
+```jsx
+import { select } from '@storybook/addon-knobs'
+import { LANGUAGE_VARIANTS } from '@bbc/psammead-storybook-helpers';
+
+const label = 'Languages';
+const defaultValue = 'This is a caption';
+const groupIdentifier = 'CAPTION VARIANTS';
+
+<Caption>
+  {select(label, LANGUAGE_VARIANTS, defaultValue, groupIdentifier)}
+</Caption>
+```
+
+### inputProvider
+
+N.B. the number of elements in the array should equal the number of arguments taken
+by the render function.
+
+```jsx
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+
+storiesOf('Caption', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    inputProvider(
+      ['caption', 'visually hidden text'],
+      (captionText, vhText) => (
+        <Caption>
+          <VisuallyHiddenText>{vhText}</VisuallyHiddenText>
+          {captionText}
+        </Caption>
+      )
+    ),
+    { notes },
+  )
+```
+
+## Contributing
+
+When **adding** a new export to this utility package the [export tests](https://github.com/bbc/psammead/blob/dab14a2732cfa620e083b7da66a148b4189474a7/packages/utilities/psammead-storybook-helpers/index.test.jsx#L13-L19) also need to be updated. When **removing** an exisiting export from this utility package the [export tests](https://github.com/bbc/psammead/blob/dab14a2732cfa620e083b7da66a148b4189474a7/packages/utilities/psammead-storybook-helpers/index.test.jsx#L13-L19) need to be updated and the package version requires a major change (EG: 1.2.1 -> 2.0.0) as this would be considered a breaking change due to functionality being removed.
```

Comment:
This bit isnt needed as you removed the test types

Reply:
This is still required since we are checking that the `LANGUAGE_VARIANTS` and `inputProvider` are correctly exported.
Rerefence [here](https://github.com/bbc/psammead/pull/378/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx#diff-b4bee899dda8b1cf31a856c214d34b26R32)

Reply:
Im unsure those tests are needed, seeing as your testing it exports them by using them

Reply:
Thanks for further clarifying Drew.
Since `LANGUAGE_VARIANTS` is being exported as well [here](https://github.com/bbc/psammead/pull/378/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx#diff-05aeb118f7f768050d0d8fbc6bbde480R4), is it okay if this [test](https://github.com/bbc/psammead/pull/378/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx#diff-b4bee899dda8b1cf31a856c214d34b26R32) is retained to check `LANGUAGE_VARIANTS` is exported correctly and remove the `inputProvider` from [this](https://github.com/bbc/psammead/pull/378/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx#diff-b4bee899dda8b1cf31a856c214d34b26R26) which is used by the aformentioned spec?